### PR TITLE
Fix XML library error and add commonly used R packages

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

**Review by KatalinaBobowik — APPROVED** [review]

File: driver/Dockerfile.r [hunk 1]
```diff
@@ -16,9 +16,9 @@ RUN apt update && \
         r-argparser \
         r-arrow \
         r-base \
+        r-biomartr \
```
Comment (KatalinaBobowik) [inline driver/Dockerfile.r:19]: do we need biomartr?

Comment (lgruen) [reply driver/Dockerfile.r:19]: We certainly don't need it. Just depends on whether we think it's used often enough to justify a slightly larger image size.

Comment (KatalinaBobowik) [reply driver/Dockerfile.r:19]: I'm just saying that `biomartr` is a different package than `biomaRt`, which I'm using in my current scrip. I haven't used `biomartr` but seems to be quite similar, albeit a different syntax

Comment (lgruen) [reply driver/Dockerfile.r:19]: Thanks! Fixed in https://github.com/populationgenomics/analysis-runner/pull/539.

**Pull request merged**